packages/react-charting/src/utilities/UtilityUnitTests.test.ts

```diff
@@ -17,7 +17,27 @@ describe('Unit test to convert data to localized string', () => {
   test('Should return undefined when data provided is undefined', () => {
     expect(utils.convertToLocaleString(undefined)).toBeUndefined();
   });
-
+  test('Should return NaN when data is NaN', () => {
+    expect(utils.convertToLocaleString(NaN)).toBeNaN();
+  });
+  test('Should return localized 0 when data is numeric 0', () => {
+    expect(utils.convertToLocaleString(0)).toBe('0');
+  });
+  test('Should return localized 123 when data is string 123', () => {
+    expect(utils.convertToLocaleString('123')).toBe('123');
+  });
```

Comment:
the sentences are not matching the return values for some tests

Reply:
Fixed statements